src/sentry/static/sentry/app/views/settings/account/accountNotificationFineTuning.jsx

```diff
@@ -122,10 +125,10 @@ class AccountNotificationsByProject extends React.Component {
             return (
               <PanelBodyLineItem key={field.name}>
                 <Select2Field
+                  defaultValue={field.defaultValue}
                   name={field.name}
                   choices={field.choices}
                   label={field.label}
-                  small={true}
```

Comment:
This would change form field to use `align-items: flex-end`, removing it will make the fields have a consistent design with our other form fields.